src/Mono.Android/Android.Runtime/OutputStreamInvoker.cs

```diff
@@ -12,6 +12,11 @@ public OutputStreamInvoker (Java.IO.OutputStream stream)
 			this.BaseOutputStream = stream;
 		}
 
+		public override void Close ()
+		{
+			BaseOutputStream.Close ();
+		}
+
 		protected override void Dispose (bool disposing)
 		{
 			if (disposing && BaseOutputStream != null) {
```

Comment:
Should we *also* call `.Close()` as part of `Dispose()` on the following line?

Reply:
Good question, I just tried this and verified that `Stream.Dispose()` calls `.Close()` for us:
```
  at Android.Runtime.OutputStreamInvoker.Close () [0x00000] in <b56c0a2d3d9045ab919839b6b74ef47d>:0 
  at System.IO.Stream.Dispose () [0x00000] in <590c33eecca549339b1b4b3b25a9dba2>:0 
  at KitKat.StorageActivity.WriteToSaf (Android.Net.Uri uri) [0x00000] in <99537faebd544453953fb4c279932f04>:0 
  at KitKat.StorageActivity.OnActivityResult (System.Int32 requestCode, Android.App.Result resultCode, Android.Content.Intent data) [0x00000] in <99537faebd544453953fb4c279932f04>:0 
  at Android.App.Activity.n_OnActivityResult_IILandroid_content_Intent_ (System.IntPtr jnienv, System.IntPtr native__this, System.Int32 requestCode, System.Int32 native_resultCode, System.IntPtr native_data) [0x00000] in <b56c0a2d3d9045ab919839b6b74ef47d>:0 
  at Android.Runtime.DynamicMethodNameCounter.7 (System.IntPtr , System.IntPtr , System.Int32 , System.Int32 , System.IntPtr ) [0x00000] in <b56c0a2d3d9045ab919839b6b74ef47d>:0 
```